README.md

```diff
@@ -98,6 +98,13 @@ The pipe's capacity for stdout/stderr can be modified by specifying the `agent.c
 to the guest kernel command line. For example, `agent.container_pipe_size=2097152` will set the stdout and stderr
 pipes to 2097152 bytes.
 
+## Agent pid namespace 
+
+The agent will allow a container to share the pid namespace of the agent when `agent.agent_pidns` is true.
```

Comment:
mmm this could be risky, any user would be able to set this through annotations, right?